plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSourceProvider.java

```diff
@@ -400,6 +402,11 @@ else if (partitionKeys.containsKey(column.getId())) {
                             trinoType,
                             deserializePartitionValue(trinoType, partitionKeys.get(column.getId()).orElse(null), column.getName()))));
                 }
+                else if (column.isPathColumn()) {
+                    columnAdaptations.add(ColumnAdaptation.constantColumn(nativeValueToBlock(
```

Comment:
I think you can inline the params. The line is not that long.

Reply:
done